### ISSUE TITLE
Fullscreen printer with two line break and style appended appears weird

### ISSUE THREAD

**Issue by edisonlee55** [Elringus/NaninovelWeb#52]
>**Describe the bug**

Fullscreen printer with two line break (`[br 2]`) and style appended (`@style #ff0000,bold`) appears weird. It has no fade animation but just shows up directly. (Watch the video in the screenshot section for a more detail view)

I am not sure if this is only happening on a printer with only two line break and a `@style #ff0000,bold` style. But I am still reporting it.

>**To Reproduce**
>1. Go to '...'
>2. Click on '....'
>3. Scroll down to '....'
>4. See error

1. Type this novel scripts into a script:
```
@printer Fullscreen
@style #ff0000,bold
Lorem ipsum dolor sit amet[br 2]consectetur adipiscing elit.
@style default
Lorem ipsum dolor sit amet[br 2]consectetur adipiscing elit.
@stop
```
2. Play that script
3. See error

>**Expected behavior**

Fullscreen printer should have fade animation.

>**Screenshots**

**Bug video (The first one is with two line break and style appended):**
https://i.gyazo.com/78a892a2a8b6a2c0fb48523fe6cc41b8.mp4

>**Naninovel version**
Version of the engine as displayed in `Naninovel -> About` window, e.g.: 1.0.12-beta

v1.4.1-beta build 2019-04-18

>**Unity version**
Specify Unity version you are using, e.g.: 2018.3.5f2.

2019.1.0f2

>**OS (when editor-related)**
Which operating system you're running the editor under, e.g.: Windows 10

macOS 10.14.4

>**Platform**
Which target platform you're using, e.g.: WebGL

PC, Mac & Linux Standalone (Mac OS X)

**Comment by Elringus** [comment]
Hm, not sure I can really spot the difference. Maybe the first word on the second line with the style applied appears a bit more "instant". Is that the issue you mean or I'm missing something?

**Comment by edisonlee55** [comment]
Yes, it's exactly the issue what I mean.

**Comment by Elringus** [comment]
Fixed in `v1.5.0-beta`.

https://i.gyazo.com/053abac558c58ca7a8561c2b1aef5d14.mp4
